src/program.js

```diff
@@ -298,6 +298,12 @@ Example: $0 --help run.
         type: 'string',
         coerce: coerceCLICustomPreference,
       },
+      'browser-console': {
+        describe: 'Use --browser-console ' +
```

Comment:
The content of `describe` becomes the help message related to the cli option in the `web-ext run --help` output, and so this can be probably just `"Open the DevTools Browser Console."`.